modules/mono/editor/bindings_generator.cpp

```diff
@@ -100,7 +100,7 @@
 
 #define BINDINGS_GENERATOR_VERSION UINT32_C(8)
 
-const char *BindingsGenerator::TypeInterface::DEFAULT_VARARG_C_IN = "\t%0 %1_in = %1;\n";
+const char*BindingsGenerator::TypeInterface::DEFAULT_VARARG_C_IN("\t%0 %1_in = %1;\n");
```

Comment:
Won't that fail `clang-format`?

Reply:
Not for the one I have. Let's see what travis has to say.

Reply:
Never mind, I just realized you were referring to `char*`. Not sure how that happened.

Reply:
Fixed.